### PR TITLE
Page Migration script

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://jira.ez.no/browse/EZEE-2150
| Versions      | 2.2, master

Based on description in https://github.com/ezsystems/ezplatform-page-migration/pull/1

### CODE REVIEW

**Comment by DominikaK** [comment]
ping @webhdx for review

File: docs/releases/updating_ez_platform.md [hunk 1]
```diff
@@ -383,10 +383,43 @@ These steps are only relevant for some releases:
...
 
-!!! caution "Updating to 2.2"
+    #### Migrating Landing Pages
 
-    To update to 2.2 with existing Content you will need a dedicated script for converting the Landing Page into the new Page.
-    [This script is currently a work in progress.](https://jira.ez.no/browse/EZEE-2150)    
+    To update to v2.2 with existing Landing Pages, you need to use a dedicated script.
+    The script is contained in a separate bundle. To use it:
+
+    1. `composer require ezsystems/ezplatform-page-migration`
```
Comment (MagdalenaZuba) [inline docs/releases/updating_ez_platform.md]: Run `composer require ezsystems/ezplatform-page-migration`

File: docs/releases/updating_ez_platform.md [hunk 2]
```diff
@@ -383,10 +383,43 @@ These steps are only relevant for some releases:
             ADD INDEX `eznotification_owner_is_pending` (`owner_id` ASC, `is_pending` ASC);
             ```
 
-!!! caution "Updating to 2.2"
+    #### Migrating Landing Pages
```
Comment (MagdalenaZuba) [inline docs/releases/updating_ez_platform.md]: Migrate

File: docs/releases/updating_ez_platform.md [hunk 3]
```diff
@@ -383,10 +383,43 @@ These steps are only relevant for some releases:
...
 
-    To update to 2.2 with existing Content you will need a dedicated script for converting the Landing Page into the new Page.
-    [This script is currently a work in progress.](https://jira.ez.no/browse/EZEE-2150)    
+    To update to v2.2 with existing Landing Pages, you need to use a dedicated script.
+    The script is contained in a separate bundle. To use it:
```
Comment (MagdalenaZuba) [inline docs/releases/updating_ez_platform.md]: is contained in ezplatform-page-migration bundle.

**Pull request merged**